src/panels/lovelace/editor/view-editor/hui-view-background-editor.ts

```diff
@@ -0,0 +1,81 @@
+import "@material/mwc-list/mwc-list-item";
+import { CSSResultGroup, LitElement, css, html, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import "../../../../components/user/ha-user-badge";
+import { LovelaceViewConfig } from "../../../../data/lovelace/config/view";
+import { HomeAssistant, ValueChangedEvent } from "../../../../types";
+import "../../../../components/ha-picture-upload";
+import type { HaPictureUpload } from "../../../../components/ha-picture-upload";
+import { CropOptions } from "../../../../dialogs/image-cropper-dialog/show-image-cropper-dialog";
+
+const cropOptions: CropOptions = {
+  round: false,
+  type: "image/jpeg",
+  quality: 0.75,
+  aspectRatio: 1.78,
+};
+
+@customElement("hui-view-background-editor")
+export class HuiViewBackgroundEditor extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _config!: LovelaceViewConfig;
+
+  set config(config: LovelaceViewConfig) {
+    this._config = config;
+  }
+
+  protected render() {
+    if (!this.hass) {
+      return nothing;
+    }
+
+    const backgroundUrlRegex = /url\(['"]?([^'"]+)['"]?\)/;
+    const backgroundUrlMatch = backgroundUrlRegex.exec(
+      this._config?.background || ""
+    );
+    const backgroundUrl = backgroundUrlMatch ? backgroundUrlMatch[1] : null;
+
+    return html`
+      <p>
+        ${this.hass.localize(
+          "ui.panel.lovelace.editor.edit_view.background.title"
+        )}
+      </p>
+      <ha-picture-upload
+        .hass=${this.hass}
+        .value=${backgroundUrl}
+        crop
+        .cropOptions=${cropOptions}
+        original
+        @change=${this._backgroundChanged}
+      ></ha-picture-upload>
+    `;
+  }
+
+  private _backgroundChanged(ev: ValueChangedEvent<string | null>) {
+    const backgroundUrl = (ev.target as HaPictureUpload).value;
+    const config = {
+      ...this._config,
+      background: backgroundUrl
```

Comment:
This is an old (and deprecated?) way of setting the background, preferably it should use themes

Reply:
Themes are not global, you set a theme on a specific view or even a card.

Reply:
It wouldn't really help with this because
1. You'd need to create a separate theme for each view. This would result in lots of duplication because I'm not aware of templating/partial overrides existing for themes.
2. There's no visual editor for themes, so discoverability would still remain an issue.

I think the current behaviour is perfect where a theme can define a default background, but a view can choose to deviate from it.